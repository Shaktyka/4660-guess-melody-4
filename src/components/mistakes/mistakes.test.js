import React from 'react';
import renderer from 'react-test-renderer';

import Mistakes from './mistakes.jsx';

describe(`Mistakes renders correctly`, () => {

  it(`Mistakes with 0 mistakes`, () => {
    const mistakes = 0;

    const tree = renderer.create(
        <Mistakes count={mistakes} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Mistakes with 2 mistakes`, () => {
    const mistakes = 2;

    const tree = renderer.create(
        <Mistakes count={mistakes} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
