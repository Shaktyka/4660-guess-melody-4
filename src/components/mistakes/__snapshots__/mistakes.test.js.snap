// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mistakes renders correctly Mistakes with 0 mistakes 1`] = `
<div
  className="game__mistakes"
/>
`;

exports[`Mistakes renders correctly Mistakes with 2 mistakes 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
